### PR TITLE
modify documentation

### PR DESCRIPTION
**Pull request by linglp**
Modify our documentation.

### CODE REVIEW

**Comment by linglp** [comment]
@milen-sage and @mialy-defelice Could you review this again? I have updated the documentation based on our code review session today. The PR related to this one is [here](https://github.com/Sage-Bionetworks/schematic/pull/592)

**Comment by linglp** [comment]
This should also help clarify the issue mentioned [here](https://github.com/Sage-Bionetworks/schematic/issues/544)

File: api/README.md [hunk 1]
```diff
@@ -7,8 +7,84 @@ source .venv/bin/activate
...
+* Step 2: Update master_fileview in config.yml
+
+    Make sure that your config.yml is pointing to the right master_fileview. For HTAN, the master fileview should be: [syn20446927](https://www.synapse.org/#!Synapse:syn20446927/tables/)
+
+* Step 3: Use parent id of the manifest for "dataset_id" parameter
+
+    Make sure that you use the parent id instead of the id of the manifest. For example, the parent Id of the biospecimen manifest that we want to obtain is: [syn24995166](https://www.synapse.org/#!Synapse:syn24995166)
+
+To sum up, use the following values to fill out the parameters in swagger UI: 
+* schema_url: https://raw.githubusercontent.com/ncihtan/data-models/main/HTAN.model.jsonld
```
Comment (milen-sage) [inline api/README.md:73]: Let's keep generic and not have references to HTAN.

**Review by milen-sage — CHANGES_REQUESTED** [review]
See comments in the review.

File: api/README.md (hunk 1 above)
Comment (linglp) [reply api/README.md]: @milen-sage thank you! The updated version is more general now. I have removed the references to HTAN.

**Review by linglp — COMMENTED** [review]
Eventually, we want to consolidate all the documentation to a single ReadMe. Here's the issue for future references: [issue 602](https://github.com/Sage-Bionetworks/schematic/issues/602)

File: api/README.md [hunk 2]
```diff
@@ -7,8 +7,72 @@ source .venv/bin/activate
...
+
+To resolve: 
+
+Make sure that the following libraries have the correct version: 
+* itsdangerous version: 1.1.0
+* jinja2 version: 2.11.3
+* markupsafe version: 2.0.1
+
+## Notes for using API endpoints: 
+Before trying out the API endpoints, please make sure that you have obtained `credentials.json`, `schematic_service_account_creds.json`, and `token.pickle`.
```
Comment (milen-sage) [inline api/README.md:43]: We can tackle this in a separate PR, but we could provide links to instructions on how to get these credentials for Google API services.

Comment (milen-sage) [inline api/README.md:43]: @linglp could you change "the API endpoints" to "schematic features utilizing Google Services (e.g. manifest generation)".

File: api/README.md [hunk 3]
```diff
@@ -7,8 +7,72 @@ source .venv/bin/activate
...
+* jinja2 version: 2.11.3
+* markupsafe version: 2.0.1
+
+## Notes for using API endpoints: 
+Before trying out the API endpoints, please make sure that you have obtained `credentials.json`, `schematic_service_account_creds.json`, and `token.pickle`. 
+
+
+###  GET /manifest/generate
+
+This endpoint functions similarly like the following:
```
Comment (milen-sage) [inline api/README.md:48]: change "like" to "to"

File: api/README.md [hunk 4]
```diff
@@ -7,8 +7,72 @@ source .venv/bin/activate
...
+Examples: 
+1) Generate a patient manifest by using the sample data model.
+
+    Simply click on "Try it out" on swagger UI and then input "None" for dataset_id parameter
+
+2) Get an existing biospecimen manifest (synapse ID: syn26543611) from HTAN TNP-TMA by using HTAN data model: 
+
+* Step 1:  Make sure you have credentials to download the desired manifest from Synapse. The "download" button should be disabled if you don't have credentials. 
+
+* Step 2: Update master_fileview in config.yml. Make sure that your config.yml point to the right master fileview.
```
Comment (milen-sage) [inline api/README.md:62]: point**s**

File: api/README.md [hunk 5]
```diff
@@ -7,8 +7,72 @@ source .venv/bin/activate
...
+    
+* For the input_token parameter, please use the value of `auth token` in your `.synapseConfig`
+
+* For the dataset_id parameter, please create a test folder on synapse and use its synapse ID
+
+* For uploading a csv file, please generate an empty manifest by using `GET /manifest/generate` endpoint and fill it out. 
+
+For the patient manifest, "Family History" column only accepts a list. The details of the validation rules (based on the example data model) could be found [here](https://github.com/Sage-Bionetworks/schematic/blob/develop/tests/data/example.model.csv)
+
+After execution, you should be able to see your manifest.csv get uploaded to your test folder on Synapse. The execution should respond the synapse ID of your submitted manifest.
```
Comment (milen-sage) [inline api/README.md:78]: "respond" to "return"

**Review by milen-sage — CHANGES_REQUESTED** [review]
It's getting good! I left a few minor change requests.

File: api/README.md (hunk 2 above)
Comment (linglp) [reply api/README.md]: Thanks Milen! I will add a link for people to obtain credentials.

**Pull request closed without merging**